src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -11,11 +11,15 @@
 
 import click
 import cloup
+from click_default_group import DefaultGroup  # type: ignore
 
 from orquestra.sdk.schema.configs import RemoteRuntime, RuntimeName
 
 from . import _cli_logs
 
+# note: click_default_group does not have stubs available for typing.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** This package hasn't been updated in 4 years. It's a single file, could we vendor it and start adding improvements.

Reply:
Vendored as of [3b0398e](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/206/commits/3b0398ecbb41bce9c5b20834a8156dff6c9f52eb), now with added Typehints!